lightly/active_learning/agents/agent.py

```diff
@@ -0,0 +1,55 @@
+from typing import *
+
+from lightly.active_learning.config.sampler_config import SamplerConfig
+from lightly.active_learning.scorers.scorer import Scorer
+from lightly.api.api_workflow_client import ApiWorkflowClient
+from lightly.api.bitmask import BitMask
+
+
+class ActiveLearningAgent:
+    """A basic class providing an active learning policy
+
+    Args:
+        api_workflow_client: The client to connect to the api
+        preselected_tag_id: The tag defining the already chosen samples (e.g. already labelled ones), default: None
+    """
```

Comment:
Classes don't have args but attributes (see [3.8.4](https://google.github.io/styleguide/pyguide.html)). Also, don't forget the newline after the colon and add punctuation
```python
Attributes:
    api_workflow_client:
        The client to connect to the api.
```